api/serializers.py

```diff
@@ -171,6 +170,15 @@ class Meta:
     embedUrl = serializers.SerializerMethodField(
         help_text='A URL suitable for embedding this media item in an IFrame', read_only=True)
 
+    downloadableByUser = serializers.BooleanField(
+        source='downloadable_by_user',
+        help_text=(
+            'Whether the current user can download this media item. '
+            'Some users can download media even if the downloadable flag is False.'
+        ),
+        read_only=True
+    )
+
```

Comment:
We may want to genericise this a bit into "capabilities" and have a media resource have a "capabilities" dict that looks like:
```js
{
    // ... resource ...
    "userCapabilities": {
        "download": true,
        "edit": true
    }
}
```
The "view" capability is, of course, implied by being able to see the resource :).

Reply:
yes, that makes it more clean